Add WinSCP support for Linux hosts

sturner88 opened the pull request:
Open WinSCP for linux hosts from context menu
![2016-06-18 22_30_50-](https://cloud.githubusercontent.com/assets/20018481/16173549/644b1218-35a4-11e6-94da-d7b41b7f1c15.png)
![2016-06-18 22_32_16](https://cloud.githubusercontent.com/assets/20018481/16173554/9811a31e-35a4-11e6-8b73-e381c33cb4f8.png)

sturner88 committed:
Add winscp for linux hosts

sturner88 committed:
Merge remote-tracking branch 'origin/dev' into feature/winscp-for-linux

# Conflicts:
#	QuickConnectPlugin/ConnectionMethodType.cs
#	QuickConnectPlugin/ConnectionMethodTypeUtils.cs
#	QuickConnectPlugin/FormOptions.Designer.cs
#	QuickConnectPlugin/QuickConnectPluginExt.cs
#	QuickConnectPlugin/QuickConnectPluginSettings.cs
#	QuickConnectPlugin/QuickConnectUtils.cs
#	README.md

sturner88 committed:
Fix forms designer merge conflict

cristianst85 commented on QuickConnectPlugin/QuickConnectPluginExt.cs:
```diff
@@ -193,6 +194,44 @@ public class QuickConnectPluginExt : Plugin {
+                                hostPwEntry.GetPassword())
+                                );
+                            cmd.StandardInput.Flush();
+                            cmd.StandardInput.Close();
```
There are a few thing that need to be done here:
- create a new class `WinSCPArgumentsFormatter` implementing `IArgumentsFormatter`
- add a test case for `WinSCPArgumentsFormatter`
- refactor using `ProcessUtils.StartDetached()`

sturner88 committed:
- Add WinSCPArgumentsFormatter

- Refactor process start method
- Add and fix test cases
- Add nuget dependencies for NUnit

sturner88 commented:
I have implemented the changes. Take a look and let me know if it's ok.

cristianst85 commented:
Looks good to me.

Now we have a non-coding issue, regarding the [copyright of the WinSCP icon](https://winscp.net/eng/docs/license#license_of_winscp_icon_set):

> Note that all images distributed in or with WinSCP application are NOT published under the GPL license. It means that it is not allowed to redistribute or reuse these images or parts of them or modifications of them without WinSCP separately or in or with another software.

I see three possible solutions:
- to ask for special permission to use/distribute the icon (highly improbable);
- find an alternative icon with a compatible license (LGPL/GPL, Creative Commons);
- or use no icon (for now at least).

P.S. This is also the reason that I've not used the original VMWare icons, etc.

sturner88 committed:
Change winscp icon

cristianst85 commented:
Thanks for your contribution!
